common/src/main/java/com/wynntils/services/discord/DiscordService.java

```diff
@@ -31,11 +32,20 @@ public DiscordService() {
         super(List.of());
     }
 
-    public void load() {
-        DiscordGameSDKCore.loadLibrary();
+    public boolean load() {
+        try {
+            DiscordGameSDKCore.loadLibrary();
+        } catch (UnsatisfiedLinkError e) {
+            if (McUtils.player() != null) {
+                McUtils.sendErrorToClient(I18n.get("service.wynntils.discord.failedToLoadSDK"));
+            }
```

Comment:
```suggestion
            McUtils.sendErrorToClient(I18n.get("service.wynntils.discord.failedToLoadSDK"));
```

This check is built-in in sendErrorToClient.